modules/margin-protocol/src/lib.rs

```diff
@@ -114,6 +120,10 @@ decl_error! {
 		TraderWouldBeUnsafe,
 		UnsafePool,
 		PoolWouldBeUnsafe,
+		NoTraderRiskThreshold,
+		CannotTraderMarginCall,
```

Comment:
```suggestion
		SafeTrader,
```

It's more informative to name the errors as the reason of failure, instead of telling them they can't do something.

Same to `CannotTraderBecomeSafe`(reuse `UnsafeTrader`).